sqlflow/client.py

```diff
@@ -87,9 +92,19 @@ def __init__(self, server_url=None):
                 raise ValueError("Can't find environment variable SQLFLOW_SERVER")
             server_url = os.environ["SQLFLOW_SERVER"]
 
-        # FIXME(tonyyang-svail): change insecure_channel to secure_channel
-        channel = grpc.insecure_channel(server_url)
-        self._stub = pb_grpc.SQLFlowStub(channel)
+        self._stub = pb_grpc.SQLFlowStub(self.newRPCChannel(server_url, ca_crt))
+
+    def newRPCChannel(self, server_url, ca_crt):
+        if ca_crt is None and "SQLFLOW_CA_CRT" not in os.environ:
```

Comment:
Shall we move this `if` to `Client. __init__` so that all the environment variable check happens there?

Reply:
Thanks, it's a good idea, how about do that in the next PR.?

Reply:
A good idea, I will do that in the next PR.